testdata/test_secret_v2_multiple.json

```diff
@@ -0,0 +1,18 @@
+{
+    "KeyIdentifier": "/projects/scuttle/locations/us/keyrings/sctl-dev/keys/sctl-testing",
+    "Version": "2",
+    "Secrets": [
```

Comment:
Ah yep, you're correct on this one. I hand crafted those json's and i used the struct properties, not the json serialized fields. 

Though it appears to have loaded fine because i didn't exactly do a great job of interrogating the objects under test, more of just a load/drop test... so to your point. Yep. somethings weird here and it smells. 

:fist_oncoming: :face_with_head_bandage: